manifests/prometheus/operations.d/000-custom-prometheus-bosh-release.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- type: replace
+  path: /releases/name=prometheus
+  value:
+    name: prometheus
+    version: 0.0.1548085118
```

Comment:
This looks to be a release build from a PR, not from master of our fork. Can we check whether that's the case, and update it to use a merged release version instead.